### PR TITLE
Draggable overlay dynamic size

### PR DESCRIPTION
**Pull request by Iamsamwhoami**

### CODE REVIEW

File: Src/Screens/ClaimsScreen/Screens/HonestyPledge.swift [hunk 1]
```diff
@@ -45,16 +48,16 @@ extension HonestyPledge: Presentable {
             slideToClaimStackView.edgeInsets = UIEdgeInsets(horizontalInset: 0, verticalInset: 20)
             slideToClaimStackView.isLayoutMarginsRelativeArrangement = true
         }
-
+        
+        bag += stackView.didLayoutSignal.map {
+            stackView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize)
+        }.debug().distinct().bindTo(viewController, \.preferredContentSize)
```
Comment (axelbacklund) [inline Src/Screens/ClaimsScreen/Screens/HonestyPledge.swift]: Remove debug()

File: Src/Screens/DashboardScreen/Viewables/PerilInformation.swift [hunk 2]
```diff
@@ -48,41 +57,16 @@ extension PerilInformation: Presentable {
...
-        }
+        containerView.addArrangedSubview(titleLabel)
 
-        viewController.view = containerView
+        let body = MarkdownText(text: description, style: .bodyOffBlack)
+        bag += containerView.addArranged(body)
+        
+        bag += containerStackView.didLayoutSignal.map { _ in
+            containerStackView.systemLayoutSizeFitting(UIScreen.main.bounds.size)
+        }.distinct().debug().bindTo(viewController, \.preferredContentSize)
```
Comment (axelbacklund) [inline Src/Screens/DashboardScreen/Viewables/PerilInformation.swift]: debug()

File: Src/Screens/DraggableOverlayScreen/DraggableOverlay.swift [hunk 3]
```diff
@@ -73,46 +93,113 @@ extension DraggableOverlay: Presentable {
...
+        
+        let ease: Ease<CGFloat> = Ease(UIScreen.main.bounds.height, minimumStep: 0.001)
+        
         func overlayCenter() -> CGFloat {
-            return view.frame.height / 2 + overshootHeight - (overlayHeight / 2)
+            var bottomPadding: CGFloat {
+                if #available(iOS 11.0, *) {
+                    return view.safeAreaInsets.bottom
+                }
+
```
Comment (axelbacklund) [inline Src/Screens/DraggableOverlayScreen/DraggableOverlay.swift]: NIT: Use same structure as the #available above

File: Src/Screens/ProfileScreen/Screens/CharityScreen/Viewables/CharityInformation.swift [hunk 4]
```diff
@@ -18,25 +18,29 @@ extension CharityInformation: Presentable {
...
+        
+        let titleLabel = MultilineLabel(value: String(key: .PROFILE_MY_CHARITY_INFO_TITLE), style: .standaloneLargeTitle)
+        bag += containerView.addArranged(titleLabel)
+        
+        let body = MarkdownText(text: String(key: .PROFILE_MY_CHARITY_INFO_BODY), style: .bodyOffBlack)
+        bag += containerView.addArranged(body)
+        
+        bag += containerStackView.didLayoutSignal.map {
+            containerStackView.systemLayoutSizeFitting(UIView.layoutFittingCompressedSize)
+        }.debug().distinct().bindTo(viewController, \.preferredContentSize)
```
Comment (axelbacklund) [inline Src/Screens/ProfileScreen/Screens/CharityScreen/Viewables/CharityInformation.swift]: debug()

**Comment by Iamsamwhoami** [comment]
@axelbacklund fixed!

**Pull request merged**